packages/venia-concept/package.json

```diff
@@ -16,7 +16,7 @@
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.phase development"
   },
   "dependencies": {
-    "@magento/peregrine": "^0.3.0",
+    "@magento/peregrine": "^0.4.0",
```

Comment:
This fixes a bug of mine from the monorepo transition that is preventing `@magento/peregrine` from being properly symlinked into `venia-concept` currently.